contracts/ColonyFunding.sol

```diff
@@ -38,37 +38,6 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
     emit TaskPayoutSet(_id, TaskRole.Worker, _token, _amount);
   }
 
-  function setAllTaskPayouts(
```

Comment:
Meh, it's so little code that it seems a waste to make a new file. I think we're justified in removing this since it's not used in the dapp, and we're planning on deprecating in-colony tasks soon enough anyway (probably in the same release).

Reply:
If in-colony tasks were deprecated in the next release, then surely we also wouldn't remove this function, only mark it as deprecated? This is also exactly the type of function we were talking about in the events meeting as perhaps being needed to smooth over the user experience as we go down that road, was it not? I strongly believe this should remain.

Reply:
Well if you feel _strongly_ :P